### PR TITLE
updated pinning for acme_unified

### PR DESCRIPTION
**Pull request by doutriaux1**
@xylar please review

### CODE REVIEW

File: acme_metapackage.bash [hunk 1]
```diff
@@ -1 +1,51 @@
...
+"ort ==${UVCDAT_VERSION}" \
+"regridpack ==${UVCDAT_VERSION}" \
+"shgrid ==${UVCDAT_VERSION}" \
+"trends ==${UVCDAT_VERSION}" \
+"zonalmeans ==${UVCDAT_VERSION}" \
+"cdp ==1.1.0" \
+"acme_diags ==v0.1b" \
+"cibots ==0.2" \
+"output_viewer ==1.2.2" \
+"xarray ==0.9.5" \
```
Comment (xylar) [inline acme_metapackage.bash]: Could you change this to xarray 0.9.6?  I'll update the Confluence page.

File: acme_metapackage.bash [hunk 2]
```diff
@@ -1 +1,51 @@
...
+"shapely  ==1.6.1" \
+"cartopy  ==0.15.1" \
+"progressbar ==2.3" \
+"matplotlib" \
+"basemap" \
+"jupyter" \
+"nb_conda" \
+"ipython" \
+"bottleneck ==1.2.1" \
+"netcdf4 ==1.2.9"
```
Comment (xylar) [inline acme_metapackage.bash]: We should add: `+"pyevtk==1.0.0`

**Comment by xylar** [comment]
@doutriaux1, looks fine to me, with 2 suggested modifications.  Would you like me to push changes or do you want to make them?

**Comment by doutriaux1** [comment]
@xylar thx let me make them and test on Linux and mac to see if it passes. Next time around you can submit PR yourself. Does that sound good?

**Comment by xylar** [comment]
That sounds great to me.  I just wasn't sure what you had in mind with the "change" part of "review/change/approve" in your comment on Confluence.

**Comment by doutriaux1** [comment]
@xylar I uploaded the conda package to uvcda torganization, do we want it on acme channel as well.

**Comment by xylar** [comment]
I didn't know there *was* and acme channel but that does sounds practical.

**Pull request merged**